aioredis/connection.py

```diff
@@ -39,6 +39,7 @@
     'PSUBSCRIBE', b'PSUBSCRIBE',
     'UNSUBSCRIBE', b'UNSUBSCRIBE',
     'PUNSUBSCRIBE', b'PUNSUBSCRIBE',
+    'PING', b'PING',
```

Comment:
This will fail then though:

```
        is_pubsub = command in _PUBSUB_COMMANDS
        if self._in_pubsub and not is_pubsub:
>           raise RedisError("Connection in SUBSCRIBE mode")
E           aioredis.errors.RedisError: Connection in SUBSCRIBE mode

aioredis/connection.py:274: RedisError
```